### PR TITLE
feat(): add new component gv-chart-gauge

### PR DESCRIPTION
**Pull request by loriepisicchio**

### CODE REVIEW

File: src/charts/gv-chart-gauge.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          borderWidth: 0,
+        }],
+      },
+
+      yAxis: {
+        min: 0,
+        max: this.max,
+        lineWidth: 0,
+        tickPositions: [],
+      },
```
Comment (gaetanmaisse) [inline src/charts/gv-chart-gauge.js:63]: yAxis looks to be an array in Highchart's doc, as it seems to work like this maybe there is an automatic wrapping

File: src/charts/gv-chart-gauge.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LitElement } from 'lit-element';
+import { ChartElement } from '../mixins/chart-element';
+
+/**
+ * Gauge chart component
+ *
+ * @attr {Array} max - The series to display on the pie chart.
```
Comment (gaetanmaisse) [inline src/charts/gv-chart-gauge.js:22]: `{number} max - The maximum value of the gauge.`

File: src/mixins/chart-element.js [hunk 3]
```diff
@@ -92,6 +94,8 @@ export function ChartElement (ParentClass) {
           Highcharts.charts[i].reflow();
         }
       };
+      ChartModuleMore(Highcharts);
+      HCSolidGauge(Highcharts);
```
Comment (gaetanmaisse) [inline src/mixins/chart-element.js:98]: Is everything still working if we move that in the constructor of `src/charts/gv-chart-gauge.js`?
Keeping them here will cause a loading of `highchart-more` + `gauge module` in every app using any Chart component.

Comment (loriepisicchio) [reply src/mixins/chart-element.js:98]: :+1:

File: src/charts/gv-chart-gauge.js [hunk 4]
```diff
@@ -15,16 +15,25 @@
...
 import { ChartElement } from '../mixins/chart-element';
+import ChartModuleMore from 'highcharts/highcharts-more';
+import Highcharts from 'highcharts';
+import HCSolidGauge from 'highcharts/modules/solid-gauge';
 
 /**
  * Gauge chart component
  *
- * @attr {Array} max - The series to display on the pie chart.
+ * @attr {Array} max - The maximum value of the gauge.
```
Comment (gaetanmaisse) [inline src/charts/gv-chart-gauge.js:25]: Still not an `Array ` 😛

Comment (loriepisicchio) [reply src/charts/gv-chart-gauge.js:25]: That would not happen if you used suggestions :wink:

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**